esm_runscripts/namelists.py

```diff
@@ -248,7 +248,7 @@ def echam_transient_forcing(config):
             if config["echam"].get("use_transient_forcing", False):
                 import pandas as pd
                 try:
-                    forcing_table = pd.open_csv(config["echam"]["transient_forcing_table"])
+                    forcing_table = pd.open_csv(config["echam"]["transient_forcing_table"], sep=";", index_col=0)
```

Comment:
@pgierz , I think there is a problem in this line, and that's what @xshi-awi is experiencing.

I tried this manually in Python, on mistral (`pd.read_csv("forcing_table", sep=";", index_col=0)`) and it tells me the following:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'pandas' has no attribute 'open_csv'
```

So it might be that there is an error here and the command is not `open_csv` but `read_csv` and the error is hidden by the `try` block. When I use `read_csv` I get:
```
>>> pd.read_csv("forcing_table.txt", sep=";", index_col=0)
              CO2           CH4           N2O   Eccentricity   Obliquity   Perihelion
 # Year                                                                              
1999     0.000487  6.779100e-07  2.064600e-07       0.018994   22.944859     294.2388
2000     0.000487  6.779100e-07  2.064600e-07       0.018994   22.944859     294.2388
2001     0.000487  6.779100e-07  2.064600e-07       0.018994   22.944859     294.2388
```
```